Dolapo47/Auto-Mart
# write test for user routes

#### What does this PR do?
- A summary of what this task is about
Write tests for user routes

#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
install mocha and chai-http
import app to test environment
test different cases for user creation and authentication

#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
run npm test in the terminal
#### What are the relevant pivotal tracker stories?
- PT Story link - (format - `[#166980708](https://www.pivotaltracker.com/n/projects/2346179)`)
